src/actions/profile/index.ts

```diff
@@ -140,3 +141,10 @@ export const updateProfileContractAddressAction = (address: string) => ({
   type: UPDATE_PROFILE_CONTRACT_ADDRESS,
   address,
 });
+
+export const getCurrentUserProfilePicture = (
```

Comment:
set
Because actions are called to set the data.
Only useSelectors are used for getting the data.